internal/faustjs.org/docs/gutenberg/tutorial/create-a-custom-block.mdx

```diff
@@ -0,0 +1,248 @@
+
+```jsx title="wp-blocks/MyCoolBlock.js" {1}
+import {gql} from '@apollo/client';
+...
+MyCoolBlock.fragments = {
+  key: `MyCoolBlockBlockFragment`,
+  entry: gql`
+    fragment MyCoolBlockBlockFragment on CreateBlockMycoolblock {
+      attributes {
+        color
+        content
+        message
+      }
+    }
+  `,
+};
+```
+We chose to attach the fragment as a property of the function component itself. This follows best practice for creating [colocated fragments](https://www.apollographql.com/docs/react/data/fragments/#creating-colocated-fragments).
+
+Before moving forward, check to ensure your block is being queried correctly in the GraphiQL IDE. You may need to use your browser's search and find method to search all the possible names under contentBlocks.
+
+<a href="/docs/img/graphiql-query-composer.png" target="_blank">
+  <img
+    src="/docs/img/graphiql-query-composer.png"
+    alt="The GraphiQL IDE is open to the Query Composer, which shows under posts, then nodes, then contentBlocks that the CreateMycoolblock is present, with a selection of attributes visible such as color, content, and message."
+  />
+</a>
+
+### 4.4 Include the block fragment into the page query string
+
+You want to include the fragment in the page query string and reference the block fragment key so that when the page resolves, the payload includes the data from that block.
+
+In this example we will use it in the single page template provided in the Faust Scaffold. You can learn more about the Faust Template Hierarchy [here](/docs/templates).
+
+```jsx {1,7,18} title="wp-templates/single.js"
+import blocks from '../wp-blocks';
+export default function Component(props) {
+...
+}
+Component.query = gql`
+...
+${blocks.MyCoolBlock.fragments.entry}
+query GetPost(
+...
+	post(id: $databaseId, idType: DATABASE_ID, asPreview: $asPreview) {
+		...
+		contentBlocks {
+			name
+			__typename
+			renderedHtml
+			id: nodeId
+			parentId
+			...${blocks.MyCoolBlock.fragments.key}
+      	}
+	}
+...
+}
+```
+
+If you navigate to the page that contains this block (here, named `localhost:3000/my-cool-block`), you should be able to inspect the properties in the console.
+
+<a href="/docs/img/contentBlocks-in-devtools.png" target="_blank">
+  <img
+    src="/docs/img/contentBlocks-in-devtools.png"
+    alt="Preview of the MyCoolBlock in the console."
+  />
+</a>
+
+## 5. Porting over the custom block's style.scss file
+
+While creating your custom block using `@wordpress/create-block`, the `style.scss` file was autogenerated in your block's plugin folder. You can include these in the global styles section of your Decoupled front-end app too, if you wish:
+
+```scss title="/MERL-742-Tutorial/styles/globals.css"
+...
+
+@font-face {
+	font-family: Gilbert;
+	src: url(/app/public/wp-content/plugins/mycoolblock/assets/gilbert-color.otf);
```

Comment:
The font url here should point to the headless instance not the WordPress instance. Ideally you want to port also any custom fonts as well because they won't be available.

Reply:
Fixed, thank you!